client/src/templates/Challenges/fill-in-the-blank/show.tsx

```diff
@@ -180,8 +180,8 @@ class ShowFillInTheBlank extends Component<
 
     const blankAnswers = blanks.map(b => b.answer);
 
-    const newAnswersCorrect = userAnswers.map(
-      (userAnswer, i) => userAnswer === blankAnswers[i]
+    const newAnswersCorrect = userAnswers.map((userAnswer, i) =>
+      userAnswer != null ? userAnswer.trim() === blankAnswers[i] : false
```

Comment:
```suggestion
    const newAnswersCorrect = userAnswers.map((userAnswer, i) =>
      userAnswer && userAnswer.trim() === blankAnswers[i]
```

We could skip the ternary and just assert truthiness.